content/documentation/rpc/worker/get_status.mdx

```diff
@@ -22,13 +22,13 @@ The workers operate similar to a threadpool.
   workers: number
   queued: number
   capacity: number
-  executing: number
+  executeing: number
```

Comment:
Did you check our RPC before making this change? This is a bug that will break our docs and make the docs not match our RPC.  https://github.com/iron-fish/ironfish/blob/f996f898a07d43397240767f8e28ede4deb4f69f/ironfish/src/rpc/routes/worker/getStatus.ts#L23